sql/2020/14_PWA/lighthouse_pwa_audits.sql

```diff
@@ -0,0 +1,37 @@
+#standardSQL
+# Get summary of all lighthouse scores for a category
```

Comment:
Yeah I get that occasionally too when querying full table.

I've made two changes:
1. Removed title as not used.
2. Reset $ to null after I've extracted the two nodes I need.

Not sure of that helps enough but didn't get it last time I ran.

It would be nice to remove the description (which is quite long) and do one join at end for that but problem is description may not exist in every run depending if that audit was actually run that time so can't depend on that. Alternative could just grab for 100 characters of description to conserve memory.

Reply:
One technique I've used on the `parse_css` queries was implement a `WHERE LENGTH(css) < 0.1 * 1024 * 1024` restriction in case there are huge results causing things to crash. Could try that here to avoid extra large LH reports.

Reply:
That seemed to do the trick (I think).